pkg/ebpf/events_pipeline.go

```diff
@@ -413,8 +411,9 @@ func (t *Tracee) processEvents(ctx context.Context, in <-chan *trace.Event) (<-c
 					logger.Debugw("False container positive", "event.Timestamp", event.Timestamp, "eventId", eventId)
 
 					// filter container policies out
-					utils.ClearBits(&event.MatchedPolicies, policiesWithContainerFilter)
-					if event.MatchedPolicies == 0 {
+					utils.ClearBits(&event.MatchedPoliciesKernel, policiesWithContainerFilter)
```

Comment:
I have sent you an offline message about atomic store/loads for the policiesWithContainerFilter (I'm adding a TODO in a comment for the ContainerFilterEnabled() Function). Also, note that protecting the policiesWithContainerFilter with atomics won't be enough when we grow
from 64 policies (u64) to 128 (2x the CPU word size). You will require
a mutex internal to ContainerFilterEnabled() logic. Either way, check
my offline message on that!

Reply:
> You will require a mutex internal to ContainerFilterEnabled() logic. Either way, check my offline message on that!

Indeed. This must be part of the API discussion as: embed and lock all policies computation when changes are required.